qdrant-landing/content/articles/discovery-search.md

```diff
@@ -0,0 +1,95 @@
+
+This is where the concept of __vector _context___ comes in. We define _context_ as a list of pairs, which in turn are made up of a positive and a negative vector. By adding a context, we can define boundaries within the vector space, to which the positive side will always be preferred over the negative side, effectively partitioning the space in which the search is performed. After the space is partitioned, we then need a _target_ to select the points that are closest to it.
+
+![Discovery search visualization](/articles_data/discovery-search/discovery-search.png)
+
+When hearing about positive and negative vectors, one might be taken back to recommendation systems. However, now these positive and negative vectors have to be provided in the shape of a context. This is inspired from the machine-learning concept of _triplet loss_, where you have three vectors: an anchor, a positive, and a negative. But in this case, the anchor would be the vector you are searching for, and the positive and negative vectors are the ones that define the context.
+
+![Triplet loss](/articles_data/discovery-search/triplet-loss.png)
+
+[__Discovery search__](#discovery-search), then, is made up of two main inputs: the query vector –or _target_– and the _context_ we just defined. However, it is not the only way to use it, you can also __only__ provide a context, which will invoke a [__Context Search__](#context-search) instead. This is useful when you want to explore the space defined by the context, but you don't have a specific target in mind.
+
+## Discovery search
+
+Let's talk about the first case: target with context.
+
+To understand why this is useful, let's take a look at a real-world example: using a cross-modal embedder like [CLIP](https://openai.com/blog/clip/) to search for images, from text __and__ images.
+CLIP is a neural network that can embed both images and text into the same vector space. This means that if you were to search for an image, you could do so by providing a text query, and vice versa. Now, take a glance at this image, and don't look at it again:
+
+![Weird dragon](/articles_data/discovery-search/weird-dragon.png)
+
+How would you look for it if you:
+
+- Don't know the name
+- Can't see it again
+- Cannot do reverse image search.
+
+A first strategy would be describing it with text, and then looking for similar images.
+
+![Googling dragon with human heads](/articles_data/discovery-search/googling-weird-dragon.png)
+
+Trial and error might work if you are clever enough to describe it well and get close enough on the selected image. If not, good luck getting out of the similarity bubble, even if you can select multiple images. Why is this hard?
+
+Turns out, cross modal encoders might not work how you expect them to. Images and text are embedded in the same space, but they are not necessarily close to each other. This means that we can create a mental model of the distribution as two separate planes, one for images and one for text.
```

Comment:
```suggestion
Turns out, multimodal encoders might not work how you expect them to. Images and text are embedded in the same space, but they are not necessarily close to each other. This means that we can create a mental model of the distribution as two separate planes, one for images and one for text.
```

There are also cross-encoders so I thought it might be confusing.